pom.xml

```diff
@@ -3,11 +3,11 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.4</version>
+        <version>3.32</version>
```

Comment:
:heart:

Reply:
And thankfully it still works with the ancient baseline of Jenkins specified here, so no need to update that most likely.